pkg/network/deviceinfo/deviceinfo.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 const (
-	MountPath = "/etc/podinfo"
+	NetworkDeviceInfoMapAnnot = "kubevirt.io/network-device-info-map"
```

Comment:
Please consider removing `map` prefix, IMO it doesn't add value.
Also if tomorrow we decide to change the annotation value format (to not be map) the name would no longer be valid.

Reply:
If we change the annotation value not to be a map it is API change. virt-launcher containers assume it is a map.